### PR TITLE
[swss test] update setup guide for swss tests

### PR DESCRIPTION
**Pull request by yxieca**
**What I did**
Update the test setup guide according to the issue I encountered lately.

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ SWSS, Redis, and all the other required components run inside a virtual switch D
     ```
     sudo modprobe team
     sudo apt install python3-pip net-tools ethtool vlan libnl-nf-3-200 libnl-cli-3-200
-    sudo pip3 install docker pytest flaky redis distro
+    sudo pip3 install docker pytest flaky redis distro dataclasses fstring
```
Comment (daall) [inline tests/README.md:39]: Can you check what version of `python3` you have? I wonder if this is some weird `python3.7` vs. `python3.5` issue, b/c I believe these are both available by default now.

Comment (yxieca) [reply tests/README.md:39]: I have Python 3.6.9.

Comment (yxieca) [reply tests/README.md:39]: Ubuntu 18.04.3 LTS

Comment (daall) [reply tests/README.md:39]: Ok, that is why then! Let's roll with this for now b/c it doesn't seem to hurt python3.8, but let's update the instructions to be more explicit about which version of python we're using later.

**Review by daall — APPROVED** [review]

**Comment by yxieca** [comment]
retest vs please

**Comment by yxieca** [comment]
retest vs please

**Comment by yxieca** [comment]
retest vs please

**Comment by liushilongbuaa** [comment]
retest vs please

**Pull request merged**